### PR TITLE
Validate input. 

### LINKED ISSUE #43: Restrict name length
**rationale** when trying to run the build script with the name `openvault-demo.wgbh-mla-test.org`, I get an error from the AWS SDK: `LoadBalancer name cannot be longer than 32 characters (Aws::ElasticLoadBalancing::Errors::ValidationError)`

This doesn't get thrown until a lot of resources have already been created. Many of those need to be deleted manually before trying again. It would be better to catch invalid names before any resources are created.

**Done when** invalid names raise exceptions before resources are created.

### PR DESCRIPTION
**Pull request by mccalluc**
@afred? fix #43.
(Wanted to avoid repeating myself, but this idiom might be weird?)

### CODE REVIEW

File: lib/util/builder.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ def build(config)
     snapshot_id = config[:snapshot_id]
     image_id = config[:image_id]
 
+    [name.length, 32].tap do |length, max|
+      fail("Name must not be longer than #{max} characters: '#{name}' is #{length}") if length > max
+    end
+
```
Comment (afred) [inline lib/util/builder.rb]: yeah... i think repeating yourself is clearer... e.g.
```ruby
max_name_length = 32
fail "Name can be no longer than #{max_name_length} characters" if name.length > max_name_length
```

**Comment by afred** [comment]
@mccalluc also.... if the `demo.` prefix is part of the ELB name, then the actual length limit is 27

**Comment by mccalluc** [comment]
@afred: this should be better?

**Comment by afred** [comment]
@mccalluc actually, looking at the load balancers that were created for `ov-qa1.wgbh-mla-test.org`, they are named `ov-qa1-wgbh-mla-test-org-a` and `ov-qa1-wgbh-mla-test-org-b`.

If the `-a` and `-b` suffixes won't change for naming ELBs, then the max length would be 30.

**Pull request merged**